test/pools/jemalloc_pool.cpp

```diff
@@ -8,8 +8,31 @@
 #include "pool.hpp"
 #include "poolFixtures.hpp"
 
+using umf_test::test;
+using namespace umf_test;
+
 auto defaultParams = umfOsMemoryProviderParamsDefault();
 INSTANTIATE_TEST_SUITE_P(jemallocPoolTest, umfPoolTest,
                          ::testing::Values(poolCreateExtParams{
                              &UMF_JEMALLOC_POOL_OPS, nullptr,
                              &UMF_OS_MEMORY_PROVIDER_OPS, &defaultParams}));
+
+TEST_F(test, metadataNotAllocatedUsingProvider) {
+    static constexpr size_t allocSize = 1024;
```

Comment:
could you add here a description of what this test does? Just copy it from commit message "This test makes sure that jemalloc does not use memory provider to allocate metadata."

Reply:
Done.